zerver/models/streams.py

```diff
@@ -177,6 +177,10 @@ class Stream(models.Model):
         ),
     }
 
+    stream_permission_group_settings_requiring_content_access = [
+        "can_add_subscribers_group",
+    ]
```

Comment:
Let's add an assert that everything here appears in the list of `GroupPermissionSetting`s; just protection against typos.